### ISSUE TITLE
Namespace warnings

### ISSUE THREAD

**Issue by klausdk** [simplesamlphp/simplesamlphp-module-casserver#9]
Since casserver is not included in simplesaml in the future, we will be using this module.

To get rid of namespace warnings we will be running the command below, until its fixed.

```
 grep -rlE "SimpleSAML_Auth_Simple|SimpleSAML_Module|SimpleSAML_Logger" \
 | xargs -n1 sed -i -e 's/SimpleSAML_Auth_Simple/SimpleSAML\\Auth\\Simple/g' -e 's/SimpleSAML_Module/SimpleSAML\\Module/g' -e 's/SimpleSAML_Logger/SimpleSAML\\Logger/g'
```

**Comment by tvdijen** [comment]
Hi @klausdk!
Thanks for reporting this.. We have to update all external modules to use namespaces, but unfortonately this requires a lot more manual labor than just running a search/replace..

**Comment by shamus13** [comment]
Fixed and will be included in release 6.1.0.
